samples/matter/bridge/src/bridged_device_types/temperature_sensor.h

```diff
@@ -0,0 +1,36 @@
+/*
+ * Copyright (c) 2023 Nordic Semiconductor ASA
+ *
+ * SPDX-License-Identifier: LicenseRef-Nordic-5-Clause
+ */
+
+#pragma once
+
+#include "bridged_device.h"
+
+class TemperatureSensorDevice : public BridgedDevice {
+public:
+	TemperatureSensorDevice(const char *nodeLabel);
+
+	int16_t GetMeasuredValue() { return mMeasuredValue; }
+	int16_t GetMinMeasuredValue() { return mMinMeasuredValue; }
+	int16_t GetMaxMeasuredValue() { return mMaxMeasuredValue; }
+	uint16_t GetTemperatureMeasurementClusterRevision() { return mTemperatureMeasurementClusterRevision; }
+	uint32_t GetTemperatureMeasurementFeatureMap() { return mTemperatureMeasurementFeatureMap; }
+
+	CHIP_ERROR HandleRead(BridgedDevice *device, chip::ClusterId clusterId, chip::AttributeId attributeId, uint8_t *buffer,
+			      uint16_t maxReadLength);
+	CHIP_ERROR HandleReadTemperatureMeasurement(TemperatureSensorDevice *device, chip::AttributeId attributeId,
+						    uint8_t *buffer, uint16_t maxReadLength);
+	CHIP_ERROR HandleWrite(BridgedDevice *device, chip::ClusterId clusterId, chip::AttributeId attributeId, uint8_t *buffer)
+	{
+		return CHIP_ERROR_UNSUPPORTED_CHIP_FEATURE;
+	}
+
+private:
+	int16_t mMeasuredValue = 20;
+	int16_t mMinMeasuredValue = -10;
+	int16_t mMaxMeasuredValue = 40;
+	uint16_t mTemperatureMeasurementClusterRevision = 1;
```

Comment:
Cluster revision and feature map could probably be `static constexpr` as they're unlikely to change at run-time.